allchblk.c

```diff
@@ -1005,6 +1005,12 @@ STATIC struct hblk *GC_allochblk_nth(size_t lb_adjusted, int k,
     return hbp;
 }
 
+# if defined(VALGRIND_TRACKING)
```

Comment:
Add: GC_ATTR_NOINLINE
I'm not sure about the order: GC_ATTR_NOINLINE GC_API or the opposite (should be checked with popular compilers - clang, gcc, msvc cl)

Reply:
Fixed